src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { DrawerProps, Minors } from './Drawer.types';
+import { AnimatePresence, motion } from 'framer-motion';
+import { withIris } from '../../utils';
+import { DrawerContainer } from './Minors/DrawerContainer';
+import { getAnimation } from './util';
+
+const DrawerComponent = ({
+  placement,
+  isOpen,
+  size = 'md',
+  // autoFocus,
+  // initialFocusRef,
+  // finalFocusRef,
+  // returnFocusOnClose,
+  // preserveScrollBarGap,
+  children,
```

Comment:
We can add a spread prop and pass it to `motion.div` to expose motion props to the parent.

```tsx
const DrawerComponent = ({
  ...,
  children,
  ...motionProps
}: DrawerProps & MotionProps) => {
  return (
    <motion.div {...motionProps}>
  )
}
```